base/doc/usrguide.tex

```diff
@@ -775,6 +775,11 @@ \subsection{Fully-expandable document commands\label{sec:ltcmd:expandable}}
     in the standard version.
 \end{itemize}
 
+When expandable commands have only |(+)m|-type arguments, the internal
```

Comment:
Worth mentioning "without any other argument modifiers nor processors"?

Reply:
Like code comment for `\@@_declare_cmd_code:Nnn`, saying "argument specification" seems more rigorous. An argument specification consists of all things, argument types, modifiers, and processors.

> also we are dealing with expandable commands.

Thus `\NewDocumentCommand` and friends need a similar paragraph, or this paragraph, after adaption, can be move to an earlier place?

Reply:
I've extended and revised the text a bit